deriv.com.conf

```diff
@@ -1,7 +1,15 @@
+map $status $cache_header {
+    default "public, max-age=7200, s-maxage=600, must-revalidate";
+    404     "max-age=0, no-cache, no-store, must-revalidate";
+    403     "max-age=0, no-cache, no-store, must-revalidate";
+    500     "max-age=0, no-cache, no-store, must-revalidate";
+}
+
 server {
     listen 80;
     server_name  _;
-
+    add_header Cache-Control $cache_header always;
```

Comment:
I *think* we said that we should not ever cache the http page itself, and this would change that.

I think if we change this, we need to understand it.

I'd like to see what the behaviour looks like, specifically:
* Is our requirement that our releases are immediately respected
* If so, I don't think that'll happen with this Cache-Control header